pkg/controller/controller_binding_test.go

```diff
@@ -1854,14 +1861,15 @@ func TestReconcileBindingWithSecretConflictFailedAfterFinalRetry(t *testing.T) {
 	assertNumberOfActions(t, actions, 1)
```

Comment:
If we keep the change of behavior when there is an error injecting credentials, then we can remove this entire test.